src/libcore/time.rs

```diff
@@ -7,6 +7,19 @@
 // <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
+#![stable(feature = "duration", since = "1.3.0")]
```

Comment:
Should the module level stability here be updated to 1.24.0 with a new feature name?

Reply:
Not sure! My assumption is that because we're simply moving what was already stable in std to core, that we should use the same feature and version. But I can update it or mark it as unstable if desired.

Reply:
The type's stability definitely shouldn't change, but its placement in libcore is new. If nothing else, it's a nice indication in rustdoc as to when it first showed up.

@aturon what do you think?

Reply:
@sfackler makes sense! considering how libstd will always show up on docs searches before libcore, this should help. I'll make this change and then wait for @aturon's response.